src/functions.jl

```diff
@@ -40,6 +40,7 @@ This function is naturally be used for single variable bounds or integrality con
 struct SingleVariable <: AbstractScalarFunction
     variable::VariableIndex
 end
+output_dimension(::SingleVariable) = 1
```

Comment:
It is already implemented for `AbstractScalarFunction` so why is this needed ?

Reply:
Oh didn't notice that. I will remove these.